api/dependency.go

```diff
@@ -67,12 +68,15 @@ func (h DependencyHandler) List(ctx *gin.Context) {
 	var list []model.Dependency
 
 	db := h.Paginated(ctx)
-	to := ctx.Query("to.id")
-	from := ctx.Query("from.id")
-	if to != "" {
-		db = db.Where("toid = ?", to)
+	from := ctx.Query("from")
```

Comment:
Why such change? Change of parameter names would break e.g. UI. A place for the change is likely the List method in binding package (add a ListFiltered(params) method which will pass the params to client.Get), but let's leave this to follow-up PR with non-basic tests like cyclic dependency test.